tests/unit/actions/configuration/configuration-workout-action-test.php

```diff
@@ -0,0 +1,372 @@
+<?php
+
```

Comment:
I think we should also add tests for the configuration-workout-route, and the workouts-route actually (perhaps create an extra issue for it?).

Reply:
Tests for conf. workout route added! Let's create an issue for the workouts route